.pre-commit-config.yaml

```diff
@@ -85,7 +85,7 @@ repos:
           - "postcss-scss@4.0.2"
           - "postcss@8.3.11"
   - repo: https://github.com/eslint/eslint
-    rev: v7.32.0
+    rev: v8.7.0
```

Comment:
Will bumping this (and the others) increase the scope of what it catches? Might be worth runnign pre-commit on the enitre repo to be sure

Reply:
I did and it does, there is 1 error and I was going to call for help with it once this fails.

Reply:
Since it doesn't cause a test failure, maybe this is fine. I'll file an issue and maybe a front-end dev can pick it up and fix it.